server/db.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// LoginUser checks bcrypt hashed passwords match in the users table
+func LoginUser(u User) bool {
+	db := GetDB()
```

Comment:
We probably want to refactor this, as it's going to stand up a DB connection on every method call, which may cause TIME_WAIT issues.

I think it's fine for now, but we should use a single object on server start and tear it down when it shuts down.

Also, as it stands now I think the connection isn't going to be reclaimed. Probably need a ```defer db.Close()``` there.